skt/reporter.py

```diff
@@ -25,6 +25,7 @@
 import requests
 
 import skt
+from skt.misc import join_with_slash, get_patch_name, get_patch_mbox
```

Comment:
If you check out the separate commits, the lines are still in the wrong order in the first two.